src/evict/evict_lru.c

```diff
@@ -2027,14 +2027,7 @@ __evict_walk_tree(WT_SESSION_IMPL *session, WT_EVICT_QUEUE *queue, u_int max_ent
         if (__wt_cache_aggressive(session))
             goto fast;
 
-        /*
-         * If the global transaction state hasn't changed since the last time we tried eviction,
-         * it's unlikely we can make progress. Similarly, if the most recent update on the page is
-         * not yet globally visible, eviction will fail. This heuristic avoids repeated attempts to
-         * evict the same page.
-         */
-        if (!__wt_page_evict_retry(session, page) ||
-          (modified && page->modify->update_txn >= conn->txn_global.last_running))
+        if (!__wt_page_evict_retry(session, page))
```

Comment:
This change makes sense to me, but I'm nervous that it's going to cause fallout (we start evicting pages with newer content than we used to, visibility bugs might come out).
Can you get a bunch of stress testing done on this branch?

Reply:
yes, sure

Reply:
I agree, now it makes more sense to me. Even I was worried about the fallout, is there a way we soak this change for a week or so in develop without dropping it to master and check the fallout?